FAQ.md

```diff
@@ -106,15 +106,36 @@ Views are buffered, so don't panic if the view counter stays the same after you
 
 ## Should I have a big server to run PeerTube?
 
-Not really. For instance, the demonstration server [https://peertube.cpy.re](https://peertube.cpy.re) has 2 vCores and 2GB of RAM and consumes on average:
- * **CPU** -> nginx ~ 20%, peertube ~ 10%,   postgres ~ 1%, redis ~ 3%
- * **RAM** -> nginx ~ 6MB, peertube ~ 120MB, postgres ~ 10MB, redis ~ 5MB
+PeerTube should run happily on a vm with 2 threads, at least 1 Gb of RAM and enough storage for videos. In terms of network, a lot will depend on which PeerTube instances you federate with.
 
-So you would need:
- * **CPU** 1 core if you don't enable transcoding, 2 at least if you enable it (works with 1 but this is really slow)
- * **RAM** 1GB
- * **Storage** Completely depends on how many videos your users will upload
+As a real life example, the PeerTube demonstration server [https://peertube.cpy.re](https://peertube.cpy.re) runs on 2 vCores and 2GB of RAM. Average consumption is:
+ * **CPU**: nginx ~ 20%, peertube ~ 10%,   postgres ~ 1%, redis ~ 3%
+ * **RAM**: nginx ~ 6MB, peertube ~ 120MB, postgres ~ 10MB, redis ~ 5MB
 
+### CPU
+Except for video transcoding, a PeerTube instance is not really cpu bound. Neither nginx, peertube, postgres nor redis require a lot of cpu. If it were only for those, once could easily get by with just one thread.
+
+You will hugely benefit from at least a second thread though, because of transcoding. Transcoding _is_ very cpu intensive. It serves two purposes on a PeerTube instance: it ensures all videos can be played optimally in the web interface, and it generates different resolutions for the same video. PeerTube support for offloading transcoding to other machines is planned, but not yet implemented. See https://github.com/Chocobozzz/PeerTube/issues/1192 .
```

Comment:
Thank you for your hint. I checked #938, but that one is about gpu based (hardware accellerated) transcoding. The relevant thing when dimensioning is offloading to different machines. That is an entirely different issue...

Reply:
Ah, sorry, I misread :)

Reply:
@markvdb I'm sorry if I misunderstand you, but #1192 is just about transcode prioritization and not offloading?

Reply:
#947 is discussing offloading

Reply:
@markvdb @kontrollanten to be clear, because an issue mentions the problem and contributors debate it doesn't mean it is on the project's roadmap. This issue in particular isn't. It doesn't mean it isn't important, just that we cannot make promises as to when/if we can allocate ressources to tackle its implementation.